[12042022 15:38:39] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 15:38:39] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 15:38:43] log /root/dionaea/src/signals.c:36-warning: sigterm_cb loop 0x7f1e6d3e9720 w 0x5588f747a230 revents 1024
[12042022 15:39:03] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 15:39:03] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 15:40:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:41:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:42:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:43:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:44:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:45:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:46:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:47:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:48:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:49:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:50:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:51:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:52:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:53:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:54:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:55:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:56:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:57:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:58:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 15:59:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:00:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:01:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:02:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:03:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:04:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:05:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:05:43] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:05:49] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:05:49] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:06:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:07:30] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[12042022 16:07:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:08:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:09:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:10:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:11:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:11:56] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:12:00] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:12:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:13:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:14:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:15:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:16:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:17:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:17:15] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:17:19] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:17:19] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:18:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:19:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:20:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:21:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:22:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:23:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:23:22] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 16:23:27] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 16:23:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 16:24:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:25:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:26:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:27:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:28:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:29:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:30:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:31:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:32:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:33:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:34:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:35:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:36:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:37:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:38:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:39:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:40:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:41:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:42:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:43:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:44:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:45:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:46:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:47:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:48:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:49:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:50:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:51:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:52:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:53:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:54:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:55:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:56:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:57:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:58:01] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[12042022 16:58:01] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[12042022 16:58:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 16:58:32] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[12042022 16:59:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:00:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:00:51] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[12042022 17:01:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:02:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:03:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 17:05:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:31:08] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:31:08] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[12042022 18:31:14] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[12042022 18:31:14] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[12042022 18:32:15] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:33:20] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:34:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:34:54] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:34:54] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:38:25] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:39:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:40:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:42:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:43:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:44:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:45:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:49:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:54:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:57:27] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 18:57:27] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 18:57:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:57:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 18:57:27] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[12042022 20:22:40] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[12042022 20:22:40] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
